c2rust-transpile/src/c_ast/mod.rs

```diff
@@ -1713,6 +1715,7 @@ impl CTypeKind {
             UInt128 => "unsigned __int128",
             Half => "half",
             BFloat16 => "bfloat16",
+            Float128 => "__float128",
```

Comment:
This is probably just used for debug output so it's okay, but it's possible we might conflate this with `_Float128` (which we don't handle yet, but should).  That said, `__float128` and `_Float128` are both IEEE 754 binary128, so us having one `CTypeKind::Float128` type is probably good enough.

Reply:
I do think it makes sense to map them both to the same underlying type, yes.